azure-pipelines/build.yml

```diff
@@ -23,29 +23,40 @@ jobs:
     displayName: 'Install Node.js'
   - script: npm ci
     displayName: 'npm ci'
-  - script: npm run build-nomaps
-    displayName: 'npm run build-nomaps'
-  - powershell: .\package.ps1
-  - task: NuGetCommand@2
+  - script: npm run build
+    displayName: 'npm run build'
+  - script: npm run webpack
+    displayName: 'npm run webpack'
+  - task: CopyFiles@2
+    displayName: 'Copy worker files to staging'
     inputs:
-      command: pack
-      packagesToPack: '$(System.DefaultWorkingDirectory)/pkg'
-      packDestination: '$(Build.ArtifactStagingDirectory)/worker'
-      ${{ if eq(parameters.IsPrerelease, true) }}:
-        buildProperties: 'prereleaseSuffix=-alpha.$(Build.BuildNumber)'
+      sourceFolder: '$(Build.SourcesDirectory)'
+      contents: |
+        dist/src/nodejsWorker.js
+        dist/src/worker-bundle.js
+        LICENSE
+        NOTICE.html
+        package.json
+        worker.config.json
+      targetFolder: '$(Build.ArtifactStagingDirectory)/worker'
+      cleanTargetFolder: true
   - script: npm prune --production
     displayName: 'npm prune --production' # so that only production dependencies are included in SBOM
   - task: ManifestGeneratorTask@0
     displayName: 'Generate SBOM for worker'
     inputs:
       BuildDropPath: '$(Build.ArtifactStagingDirectory)/worker'
-      BuildComponentPath: '$(System.DefaultWorkingDirectory)/node_modules'
+      BuildComponentPath: '$(Build.SourcesDirectory)/node_modules'
```

Comment:
Why is this working of the `SourcesDirectory`? I thought we had just copied the files to "staging"?

Reply:
Just to be clear, `System.DefaultWorkingDirectory` and `Build.SourcesDirectory` are the same, I just prefer the shorter name

To answer your question: Once we webpack the code, anything we used from the "node_modules" folder is now in that webpacked file (worker-bundle.js) and is pretty mangled up. The sbom manifest tool can't determine the components from the webpacked file in the staging dir, so I'm referencing the original un-webpacked modules in the sources directory

Reply:
Makes sense. Do you mind documenting this in the YAML itself? It's not immediately obvious, at least to me.